examples/sgp4Prop/fsw/eci_inc/eci_interface.h

```diff
@@ -140,6 +143,20 @@ static const ECI_Evs_t ECI_Events[] = {
     (double*)&line2Check.expected,  
     0,  0,  0   
   },
+
+  { ECI_EVENT_2_DATA,
+    &timeOutOfRangeID,
+    &errorEventType,
+    &filterOneEvent,
+    &propState.invalid,
+    "FSW time outside TLE range ",
```

Comment:
```suggestion
    "FSW time (%f) outside TLE range (%f)",
```

This is a C-style format string, so you need to specify where you want `propState.t `and `propState.epoch` to be placed in the string.